src/app/parcels/ActionBar/ActionModals/DriverOverviewModal.tsx

```diff
@@ -50,7 +50,7 @@ const DriverOverviewInput: React.FC<DriverOverviewInputProps> = ({
 };
 
 const getPdfErrorMessage = (error: DriverOverviewError): string => {
-    let errorMessage: string;
+    let errorMessage: string = "";
```

Comment:
What does this do?

Reply:
The linter was complaining about it but actually I think my later changes avoided that error anyway. I'll revert this :)